docs/source/deploying-volttron/multi-platform/multi-platform-rabbitmq-deployment.rst

```diff
@@ -165,8 +165,8 @@ upstream servers on the downstream server and make the VOLTTRON exchange
 
         .. code-block:: bash
 
-            vcfg --rabbitmq federation [optional path to rabbitmq_federation_config.yml
-            containing the details of the upstream hostname, port and vhost.]
+            vcfg rabbitmq federation [--config optional path to rabbitmq_federation_config.yml
+            containing the details of the upstream hostname, port and vhost.] [--max-retries optional maximum CSR retry attempt]
 
 
         Example configuration for federation is available
```

Comment:
Remove steps to create user manually after running vcfg rabbitmq federation or shovel